lib/Runtime/Base/AuxPtrs.h

```diff
@@ -165,15 +165,16 @@ namespace Js
     }
 
     template<class T, typename FieldsEnum>
-    void AuxPtrs<T, FieldsEnum>::AllocAuxPtrFix(T* _this, uint8 size, Recycler* recycler)
+    void AuxPtrs<T, FieldsEnum>::AllocAuxPtrFix(T* pHost, uint8 size, Recycler* recycler)
```

Comment:
nit: imo, should just be `host`... `pHost` is pretty non-standard notation for our codebase.